src/provider/CentralServerProvider.tsx

```diff
@@ -514,15 +514,13 @@ export default class CentralServerProvider {
     return result.data;
   }
 
-  public async getChargingStation(id: string): Promise<ChargingStation> {
+  public async getChargingStation(params: Record<string, any>): Promise<ChargingStation> {
```

Comment:
The ID should stay in the method's signature and you can add an extra param as optional with the type used in the backend (HttpChargingStationRequest).

Reply:
I don't understand, this type only has an ID field

Reply:
Yes, you're right, you can use the ev-server's type in ev-mobile and remove the ID.
I like to see the mandatory params visible in the method's signature (the ID here.)

@Ninjeneer, we should also do the same in Angular central-server.service class.
So the types are aligned with the backend:
`getChargingStation(id: string, params?: HttpChargingStationRequest) {...}`
What do you think?

Reply:
I find it a bit confusing to call the second param 'params', as the ID is also part of the HTTP request params

Reply:
> I find it a bit confusing to call the second param 'params', as the ID is also part of the HTTP request params

This is what I explain above, remove the ID in the front-end type (the backend has all but not the fronts)
We can have a call?

Reply:
Yes for a call

Reply:
As discussed with Serge, let's use getChargingStation(id: string, extraParams?: HttpChargingStationRequest) {...}